pkgs/development/python-modules/python-can-isotp/default.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  autopep8,
+  coverage,
+  ipdb,
+  mypy,
+  pytestCheckHook,
+  python-can,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "python-can-isotp";
+  version = "2.0.6";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "pylessard";
+    repo = "python-can-isotp";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-wfZMVfLBdYkFbb0DiDWmGaraykJ/QL64Zkl2/nBu4lY=";
+  };
+
+  build-system = [ setuptools ];
+
+  passthru.optional-dependencies = {
+    dev = [
+      autopep8
+      coverage
+      ipdb
+      mypy
+      python-can
+    ];
+    test = [
+      coverage
+      mypy
+      python-can
+    ];
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ] ++ passthru.optional-dependencies.test;
+
+  pythonImportsCheck = [ "isotp" ];
+
+  meta = with lib; {
+    description = "A Python package that provides support for ISO-TP (ISO-15765) protocol";
```

```suggestion
    description = "Python package that provides support for ISO-TP (ISO-15765) protocol";
```
> Not start with definite (“The”) or indefinite (“A”/“An”) article.


https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook-using-pytestcheckhook